lib/index.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const _logger = require('./logger');
+const bedrock = require('bedrock');
+const brPackageManager = require('bedrock-package-manager');
+const {util: {BedrockError}} = bedrock;
+
+// load config defaults
+require('./config');
+
+/**
+ * @module bedrock-kms
+ */
+
+exports.callMethod = async ({request}) => {
+
+  // request is a document with an ocapld proof
+
+  // validate the request document
+
+  const plugin = 'foo'; // extracted from key id which comes out of request
+  const method = 'bar'; // method is somewhere in request
+  const controller = 'fooController'; // controller IS the keyID?
+  const keyId = controller; // are controller and keyID truely synonymous?
```

Comment:
`controller` and `keyId` are not the same thing, no. The `keyId` refers to the key that is to be used to perform the requested operation. The `controller` refers to the entity that controls that key. That entity may itself be a key or it may be something like a DID subject.